bin/animation_assets/manifest_builder.rb

```diff
@@ -194,16 +195,17 @@ def get_animation_strings
     return strings.sort.to_h
   end
 
+  def initial_animation_metadata
+    @initial_animation_metadata ||= build_animation_metadata(animation_objects, {})
+  end
+
   # Takes in a locale (for the file suffix) and a map of string translations
   # and replaces the aliases of the aliases of the animations with their translation.
   # Then, the localized manifest is uploaded to S3 with the suffix .{locale}.json
   def upload_localized_manifest(locale, strings)
     return unless upload_spritelab_to_s3?
 
-    @bucket ||= Aws::S3::Bucket.new(DEFAULT_S3_BUCKET)
-    @animation_objects ||= get_animation_objects(@bucket)
-
-    animation_metadata = build_animation_metadata(@animation_objects, {})
+    animation_metadata = initial_animation_metadata
```

Comment:
Why did we switch from `build_animation_metadata` to `initial_animation_metadata`?

Reply:
We don't switch, it's a completely new method that calls `build_animation_metadata` with some specific arguments.
`initial` means we call `build_animation_metadata` with the empty `previous_metadata` arg

Reply:
`build_animation_metadata`: `build_` is an action which means on every call of the method we get freshly generated data based on the provided arguments
`initial_animation_metadata`: means the method returns the same preserved data (initial animation metadata) on each call